client/src/components/layouts/global.css

```
@@ -615,7 +615,7 @@ blockquote .small {
   color: #31708f;
   border-color: #31708f;
 }
-.alert .btn:hover,
+
```

```suggestion
.alert .btn:hover,
```
I might be a bit confused, but this one doesn't seem to be duplicated.

---

nice catch, you are totally right